lib/routes/routeBackbeat.js

```diff
@@ -90,8 +90,14 @@ function _checkMultipleBackendRequest(request, log) {
         return errors.BadRequest.customizeDescription(errMessage);
     }
     if ((operation === 'puttagging' || operation === 'deletetagging') &&
```

Comment:
The following structure would be slightly better, grouping all tests related to a subset of operations:

```
if (operation === 'puttagging' || operation === 'deletetagging') {
    if (A) {}
    if (B) {}
}
```

Reply:
Updated.